profiles/standard/config/layout.layout.home.json

```diff
@@ -0,0 +1,184 @@
+{
+    "_config_name": "layout.layout.home",
+    "path": "home",
+    "name": "home",
+    "title": "Home page",
+    "description": null,
+    "module": null,
+    "weight": 0,
+    "storage": 1,
```

Comment:
"storage" should be set to `4`, the constant for `LAYOUT_STORAGE_DEFAULT` if we set the module name above.